otx/mpa/seg/semisl/stage.py

```diff
@@ -11,17 +11,17 @@
 logger = get_logger()
 
 
-class SemiSegStage(SegStage):
+class SemiSLSegStage(SegStage):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
```

Comment:
since `SemiSLSegStage` inherits `SegStage`, doesn't it work well without `init`?

Reply:
yeah working well. it already had init file previously but with empty content so didn't have to make it.